src/components/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,167 @@
+<!--
+  -
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  - @copyright Copyright (c) 2019 Greta Doci <gretadoci@gmail.com>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  - @author Greta Doci <gretadoci@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<docs>
+## One checkbox
+
+```vue
+<template>
+		<Checkbox
+			label="some text"
+			v-model="checkedTarget"/>
+</template>
```

Comment:
checkedTarget is undefined

```suggestion
<template>
		<Checkbox
			label="some text"
			v-model="checkedTarget"/>
</template>
<script>
 export default {
 	data() {
 		return {
 			checkedTarget: false
 		}
 	},
 }
</script>
```